packages/coqide/coqide.8.12.0/opam

```diff
@@ -0,0 +1,50 @@
+opam-version: "2.0"
```

Comment:
Let's remember to split CoqIDE out if there are problems in CI.

Reply:
I don't follow... we want a CoqIDE package, no?
I consider creating two PRs and tracking their mutual dependencies to be just unnecessary work.

Reply:
Your call; I was just mentioning @palmskog's suggestion to avoid holding up Coq packages (IIRC because coqide held up some 8.11.* version for a long while). No strong opinion myself.

Reply:
But again, no action needed if there are no problems in CI 🙂